### DISCUSSION TITLE
How can I add 'Edit this page' in the sidebar?

### DISCUSSION THREAD

**Discussion by mdobydullah** [ZTL-UwU/shadcn-docs-nuxt#48] [Q&A] [answered]
First, thank you for creating such an awesome documentation template!

I want to add the 'Edit this page' option in the sidebar, similar to Nuxt UI Pro Docs. How can I achieve this?

![image](https://github.com/user-attachments/assets/62271838-8dcd-4be0-a86c-8e29e57f7146)

**Comment by ZTL-UwU** [comment, marked as answer]
Added in #49
